exercises/custom-set/custom_set_test.rb

```diff
@@ -3,114 +3,303 @@
 require 'minitest/autorun'
 require_relative 'custom_set'
 
+# rubocop:disable Metrics/LineLength
+
+# Test data version:
+# cdfb4a2
 class CustomSetTest < Minitest::Test
-  def test_equal
-    assert_equal CustomSet.new([1, 3]), CustomSet.new([3, 1])
-    refute_equal CustomSet.new([1, 3]), CustomSet.new([3, 1, 5])
-    refute_equal CustomSet.new([1, 3, 5]), CustomSet.new([3, 1])
-    refute_equal CustomSet.new([1, 3]), CustomSet.new([2, 1])
+  def test_sets_with_no_elements_are_empty
+    # skip
+    set = CustomSet.new []
+    assert_equal CustomSet.new, set
+  end
+
+  def test_sets_with_elements_are_not_empty
+    skip
+    set = CustomSet.new [1]
+    refute_equal CustomSet.new, set
+  end
+
+  def test_nothing_is_contained_in_an_empty_set
+    skip
+    set = CustomSet.new []
+    element = 1
+    refute set.member? element
+  end
+
+  def test_when_the_element_is_in_the_set
+    skip
+    set = CustomSet.new [1, 2, 3]
+    element = 1
+    assert set.member? element
+  end
+
+  def test_when_the_element_is_not_in_the_set
+    skip
+    set = CustomSet.new [1, 2, 3]
+    element = 4
+    refute set.member? element
+  end
+
+  def test_empty_set_is_a_subset_of_another_empty_set
+    skip
+    set1 = CustomSet.new []
+    set2 = CustomSet.new []
+    assert set2.subset? set1
+  end
+
+  def test_empty_set_is_a_subset_of_non_empty_set
+    skip
+    set1 = CustomSet.new []
+    set2 = CustomSet.new [1]
+    assert set2.subset? set1
+  end
```

Comment:
These subset tests seem backwards to me. I read `set2.subset? set1` as Is set2 at subset of set1?
Looking at Ruby's `Set` class, I see it also handles it this way.